rgw/v2/tests/s3_swift/reusables/bucket_notification.py

```diff
@@ -234,12 +234,15 @@ def get_bucket_notification(rgw_s3_client, bucketname, empty=False):
     )
 
 
-def verify_event_record(event_type, bucket, event_record_path, ceph_version):
+def verify_event_record(event_type, bucket, event_record_path, ceph_version, config):
     """
     verify event records
     """
     if os.path.getsize(event_record_path) == 0:
-        raise EventRecordDataError("event record not generated! File is empty")
+        if config.test_ops.get("test_delete_object_sync_archive", False):
```

Comment:
@viduship as per the above change in method arguments, please modify this as below
if config and config.test_ops.get("test_delete_object_sync_archive", False):

Reply:
done